src/Samples/SarifToCsv/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using AzureDevOpsCrawlers.Common.IO;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Newtonsoft.Json;
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace SarifToCsv
+{
+    /// <summary>
+    ///  SarifToCsv is a simple example of converting a few key SARIF result properties to CSV format
+    ///  to allow easily viewing results compared with the 'match-results-forward' Sarif Multitool mode.
+    /// </summary>
+    /// <remarks>
+    ///  Should be modified to:
+    ///    - Make the output columns configurable.
+    ///    - Enable writing more result properties.
+    ///    - Integrate as a Sarif Multitool mode.
+    /// </remarks>
+    class Program
```

Comment:
>class [](start = 4, length = 5)

another nit, make accessibility explicit

Reply:
Fixed. This is the default in the current VS project templates.